src/renault_api/client.py

```diff
@@ -0,0 +1,70 @@
+"""Client for Renault API."""
+import logging
+from typing import Dict
+from typing import Optional
+
+import aiohttp
+from aiohttp.client_exceptions import ClientResponseError
+
+from .const import AVAILABLE_LOCALES
+from .const import CONF_GIGYA_APIKEY
+from .const import CONF_GIGYA_URL
+from .const import CONF_KAMEREON_APIKEY
+from .const import CONF_KAMEREON_URL
+from .exceptions import RenaultError
+
+_LOGGER = logging.getLogger(__package__)
+
+
+class RenaultClient:
+    """Proxy to the Renault API."""
+
+    def __init__(self) -> None:
+        """Initialise Renault Client."""
+        self.aiohttp_session: Optional[aiohttp.ClientSession] = None
```

Comment:
For now, it is not compulsory. It is only used in case the Locale is not available.
In the second step, we will probably start with PyZE which also doesn't need aiohttp client.